app/utils/markdown.js

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
-import {Platform, StyleSheet} from 'react-native';
+import {Dimensions, PixelRatio, Platform, StyleSheet} from 'react-native';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
 export function getCodeFont() {
     return Platform.OS === 'ios' ? 'Menlo' : 'monospace';
 }
 
+const scale = Dimensions.get('window').width / 320;
```

Comment:
can you use the value of `deviceWidth` in the redux store, or maybe do this dynamically as this will not be recalculated if the device rotates

Reply:
I don't think we actually want this recalculated since that would mean the font size changes when the device is rotated which would be weird.

I do however think we should figure out if there's a way to get the actual device width in case the app is opened in landscape